src/components/Author/index.tsx

```diff
@@ -12,6 +13,13 @@ const Author = ({
   username,
   size = '64',
 }: Props): null | JSX.Element => {
+  const intl = useIntl();
+
+  const translate = defineMessages({
```

Comment:
Very interesting. You actually don't need to to use the `defineMessages`. You could have done a direct call for:

`intl.formatMessage({ id: "components.author.githubLinkLabel" }, { username })`